pkgs/by-name/in/intel-npu-firmware/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  stdenvNoCC,
+  lib,
+  fetchFromGitHub,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "intel-npu-firmware";
+  version = "1.13.0";
+
+  src = fetchFromGitHub {
+    owner = "intel";
+    repo = "linux-npu-driver";
+    rev = "bd414f6905d587bf4dd49a8f74511b7a5fd6f226";
+    sha256 = "ChiVnZ7SiiW5UIjYmvj9cf1bgS+GPHX279Q9DIA11og=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/lib/firmware/intel/vpu
+    cp -P firmware/bin/*.bin $out/lib/firmware/intel/vpu
+
+    mkdir -p $out/share/doc/intel-npu-driver
+    cp LICENSE.md $out/share/doc/intel-npu-driver/LICENSE
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
```

Comment:
Also we are trying to reduce the usage of `with lib;`

see https://github.com/NixOS/nixpkgs/issues/208242